src/test/resources/testscripts/NextflowRnaseqWorkflow.sh

```diff
@@ -48,6 +11,7 @@ terra nextflow -version
 #      google.lifeSciences.network = 'network'
 #      google.lifeSciences.subnetwork = 'subnetwork'
 mv rnaseq-nf/nextflow.config rnaseq-nf/nextflow.config_original
+resourceName="terraclitesting"
```

Comment:
Since same value of resourceName should be used in setup & workflow scripts, might be easier to maintain if it is passed as an arg in Nextflow.java